fendermint/vm/topdown/src/sync/pointers.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright 2022-2023 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use crate::BlockHeight;
+use std::fmt::{Display, Formatter};
+
+/// There are three pointers, each refers to a block height, when syncing with parent. As Lotus has
```

Comment:
This comment is redundant with the one below. Maybe remove this one and keep the one from `sync.rs`?